src/marlin_stubs/pause/pause.cpp

```diff
@@ -52,6 +52,9 @@
 #include "filament.h"
 #include "RAII.hpp"
 
+#define MINIMAL_PURGE                   1
```

Comment:
please avoid using #define in C++ code, use enum or static const something etc.

Reply:
fixed